public/js/admin/admin.js

```
@@ -1 +1,528 @@
+
+    $('[data-setting-type]').on('click', function () {
+        // Get the value of the `data-setting-type` attribute
+        var settingType = $(this).data('setting-type');
+
+        // Perform your action here
+        console.log('Setting type clicked:', settingType);
+        // You can add more actions here
+        switch (settingType) {
+            case 'email-notification-version':
+
+                var state = $(this).data('value');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)